src/wasm-stack.h

```diff
@@ -144,6 +152,8 @@ class BinaryInstWriter : public OverriddenVisitor<BinaryInstWriter> {
   WasmBinaryWriter& parent;
   BufferWithRandomAccess& o;
   Function* func = nullptr;
+  bool sourceMap;
```

Comment:
is this just use for source maps for for dwarf debug_line too?

Reply:
This is just for source maps - when source maps are enabled, we don't update debug info in the way that DWARF needs, and vice versa. So we need to know if source maps are used (previously there was just one way to do debug locations).